src/test/resources/out/compare/ModuleDeviation.json

```diff
@@ -0,0 +1,307 @@
+            "name": "type",
+            "description": "",
+            "config": true,
+            "class": "leaf",
+            "status": "CURRENT"
+          },
+          {
+            "path": "/mo:data-container/mo:list-data/mo:sub-device/mo:year/",
+            "type_info": {
+              "description": "",
+              "type": "uint8"
+            },
+            "children": [],
+            "name": "year",
+            "description": "",
+            "config": true,
+            "class": "leaf",
+            "status": "CURRENT"
+          },
+          {
+            "path": "/mo:data-container/mo:list-data/mo:sub-device/mo:state/",
+            "type_info": {
+              "description": "",
+              "type": "string"
+            },
+            "children": [],
+            "name": "state",
+            "description": "",
+            "config": true,
+            "class": "leaf",
+            "status": "CURRENT"
+          }
+        ],
+        "name": "sub-device",
+        "description": "",
+        "config": true,
+        "class": "list",
+        "status": "CURRENT"
+      }],
+      "name": "/mo:data-container/mo:list-data/",
+      "description": "",
+      "config": true,
+      "class": "augmentation",
+      "status": "CURRENT"
+    },
+    {
+      "path": "/mo:data-container/",
+      "children": [{
+        "path": "/mo:data-container/mo:device/",
+        "type_info": {},
+        "children": [{
+          "path": "/mo:data-container/mo:device/mo:enumType/",
+          "type_info": {
+            "description": "",
+            "type": "mo:enumeration"
+          },
+          "children": [],
+          "name": "enumType",
+          "description": "",
+          "config": true,
+          "class": "leaf",
+          "status": "CURRENT"
+        }],
+        "name": "device",
+        "description": "",
+        "config": true,
+        "class": "container",
+        "status": "CURRENT"
+      }],
+      "name": "/mo:data-container/",
+      "description": "",
+      "config": true,
+      "class": "augmentation",
+      "status": "CURRENT"
+    },
+    {
+      "path": "/mo:data-container/mo:device/",
+      "children": [{
+        "path": "/mo:data-container/mo:device/mo:model/",
+        "type_info": {
+          "description": "",
+          "type": "string"
+        },
+        "children": [],
+        "name": "model",
+        "description": "",
+        "config": true,
+        "class": "leaf",
+        "status": "CURRENT"
+      }],
+      "name": "/mo:data-container/mo:device/",
+      "description": "",
+      "config": true,
+      "class": "augmentation",
+      "status": "CURRENT"
+    },
+    {
+      "path": "/mo:data-container/mo:device/",
+      "children": [{
+        "path": "/mo:data-container/mo:device/mo:manufacture/",
+        "type_info": {
+          "description": "",
+          "type": "string"
+        },
+        "children": [],
+        "name": "manufacture",
```

Comment:
IMO: probably it should be `manufacturer`.

Reply:
Fixed